src/services/videos-services.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const getVidoes = async () => {
+    try{
+        return await axios.get('/api/videos');
+    }catch{
+        console.log('Couldn\'t get vidoes.');
```

Comment:
we should remove console.log before commit.

Reply:
Noted. Thanks!